Fixes issue where KeyRotationScheduler isn't LifecycleAware

ccunniff opened the pull request:
https://github.com/libertymutual/ssh-key-enforcer-stash/issues/29

ccunniff committed:
Fixes issue where KeyRotationScheduler isn't LifecycleAware

                                           https://github.com/libertymutual/ssh-key-enforcer-stash/issues/29

jdesulme commented:
It would be good if we could add some kind of test to make sure this doesn't fail in the future... And we don't know about it.

ccunniff committed:
Tests that key rotation job is scheduled

removes dysfunctional clusterTestGroup while using runCluster.sh

https://github.com/libertymutual/ssh-key-enforcer-stash/issues/29

ccunniff committed:
Uses local postgres db for testing data center cluster instead of random bricked db

https://github.com/libertymutual/ssh-key-enforcer-stash/issues/29

jdesulme commented:
Nice use of docker to spin up the db locally!
